wenet/transformer/asr_model.py

```diff
@@ -433,6 +434,101 @@ def _ctc_prefix_beam_search(
         hyps = [(y[0], log_add([y[1][0], y[1][1]])) for y in cur_hyps]
         return hyps, encoder_out
 
+    def _ctc_prefix_beam_search_with_bias(
```

Comment:
Could we add context_graph directly in _ctc_prefix_beam_search?

Reply:
Yes, I directly add the context_graph into _ctc_prefix_beam_search in the new commit